texar/modules/encoders/bert_encoders.py

```diff
@@ -120,105 +120,105 @@ def default_hparams():
         .. code-block:: python
 
             {
-            'pretrained_model_name': 'bert-base-uncased',
-            'embed': {
-                'dim': 768,
-                'name': 'word_embeddings'
+            "pretrained_model_name": "bert-base-uncased",
```

Comment:
This part is Python code, so it should be formatted as Python code. The contents of this bracket should be further indents. This also applied to the `BertClassifier` example.